packages/mgt/src/components/mgt-agenda/mgt-agenda.ts

```diff
@@ -259,6 +259,15 @@ export class MgtAgenda extends MgtTemplatedComponent {
     `;
   }
 
+  /**
+   * Reloads the component with its current settings and potential new data
+   *
+   * @memberof MgtAgenda
+   */
+  protected async reload() {
```

Comment:
:)

```suggestion
  public async reload() {
```

Reply:
Seriously? I think I'm too tired. I need to go to bed haha!